### ISSUE TITLE
teensy 4 version

### ISSUE THREAD

**Issue by pppier** [mxmxmx/O_C#119]
hello
you know you did an amazing work with ornament and crime.
now teensy 3.2 is not available, would you consider doing a teensy4 verision of the official (deep) firmware? 
same for temps utile, it would be a great chance to still have it available
surely an update implementing teensy4 new possibilities (oc and/or tu) would be even better... midi, save, more memory so more apps, audio out, and so on ... but even a t4 porting would be great, what is your opinion?
bests 
p

**Comment by patrickdowling** [comment]
Yeah, this comes up a lot, see e.g. https://github.com/mxmxmx/O_C/issues/110 and various MW threads.

The short answer (for me at least) is that it’s not that interesting, but I haven't stumbled on some "killer app" or other Really Good Reason to pursue it. The faster processor doesn’t solve the underlying hardware limitation of DAC & screen on the same port — which IMO precludes serious audio use — and the T4.0 has even less non-volatile memory unless you want to jump through some hoops.

That said curiosity got the better of me and I did eventually write some [drivers for the T4](https://github.com/patrickdowling/oCt4drv), but porting the old apps will be somewhat tedious.
